src/components/ui/PaymentRequestTable/PaymentRequestTable.tsx

```diff
@@ -9,6 +9,9 @@ interface PaymentRequestTableProps {
   pageSize: number;
   totalRecords: number;
   onPaymentRequestClicked?: (paymentRequest: LocalPaymentRequest) => void;
+  onPaymentRequestDuplicateClicked?: (paymentRequest: LocalPaymentRequest) => void;
+  onPaymentRequestDeleteClicked?: (paymentRequest: LocalPaymentRequest) => void;
+  onPaymentRequestCopyLinkClicked?: (paymentRequest: LocalPaymentRequest) => void;
```

Comment:
Maybe we can make all of these required? I don't see a reason why they should be optional now.